ProjectTemplates/templates/NetCord.GenericHost/NetCord.Template.Bot.csproj

```
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+<!--#if (framework == "net9.0") -->
+        <TargetFramework>net9.0</TargetFramework>
+<!--#else -->
+        <TargetFramework>net8.0</TargetFramework>
+<!--#endif -->
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+    </PropertyGroup>
+
+    <ItemGroup>
+<!--#if (framework == "net9.0") -->
+        <PackageReference Include="Microsoft.Extensions.Hosting" Version="9.0.5" />
+<!--#else -->
+        <PackageReference Include="Microsoft.Extensions.Hosting" Version="8.0.1" />
+<!--#endif -->
+        <PackageReference Include="NetCord.Hosting" Version="1.0.0-alpha.380" />
+        <PackageReference Include="NetCord.Hosting.Services" Version="1.0.0-alpha.380" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <Content Update="appsettings.json">
```

You could make this a cleaner single line entry:
```suggestion
        <Content Update="appsettings.json" CopyToOutputDirectory="PreserveNewest" />
```